tasks/helpers/deployHelpers.ts

```diff
@@ -136,7 +136,7 @@ export async function deployContract(
     `yarn hardhat verify ${contractAddress} --network ${hre.network.name} ${proxyAddress ? '' : constructorArgs.join(' ')}`,
   )
 
-  return contractAddress || proxyAddress
+  return proxyAddress || contractAddress
```

Comment:
Returning the proxy first as it's usually what we want.
But let me know @jophish if there's any issue with that.